docs/source/examples/zne-braket-ionq.myst

```diff
@@ -0,0 +1,239 @@
+By default, noise is scaled by locally folding gates at random, and the default extrapolation is Richardson.
+
+To specify a different extrapolation technique, we can pass a different {class}`.Factory` object to {func}`.execute_with_zne`. 
+The following code block shows an example of using linear extrapolation with five different (noise) scale factors.
+Moreover, instead of _local unitary folding_, _global unitary folding_ is used to scale noise.
+More details on ZNE options are given [here](../guide/zne-3-options.myst).
+
+```{code-cell} ipython3
+factory = zne.inference.LinearFactory(scale_factors=[1.0, 1.5, 2.0, 2.5, 3.0])
+noise_scaling_method = zne.scaling.fold_global
+
+mitigated = zne.execute_with_zne(
+    braket_circuit,
+    braket_ionq_execute,
+    factory=factory,
+    scale_noise=noise_scaling_method, 
+)
+print(f"Mitigated result {mitigated:.3f}")
+```
+
+Let's visualize the zero-noise extrapolation fit.
+
+```{code-cell} ipython3
+_ = factory.plot_fit()
+```
+
+Any different combination of noise scaling and extrapolation technique can be passed as arguments to
+{func}`.execute_with_zne`.
+
+
+## Lower-level usage
+
+Here, we show a more detailed usage of the Mitiq library which mimics what happens in the call to
+{func}`.execute_with_zne` used in the previous sections. This low-level approach allows us to have
+a better control of the error mitigation workflow.
+
+First, we define factors to scale the circuit length by and then we fold the circuit using the {func}`.fold_global` method.
+
+```{code-cell} ipython3
+scale_factors = [1., 1.5, 2., 2.5, 3.]
+folded_circuits = [
+        zne.scaling.fold_global(braket_circuit, scale)
+        for scale in scale_factors
+]
+
+# Check that the circuit depth is (approximately) scaled as expected
+length_in = len(braket_circuit.instructions)
+for j, c in enumerate(folded_circuits):
+    length_out = len(c.instructions)
+    print(f"Number of gates of folded circuit {j} scaled by: {length_out / length_in:.3f}")
+```
+
+The number of gates has been scaled to approximate the input ``scale_factors``.
+
++++
+
+For a noiseless simulation, the expectation value should be 1.0 because our circuit compiles to the identity.
+For a noisy simulation, the value will be smaller than one. Because folding introduces more gates and thus more noise,
+the result will decrease as the length of the folded circuits increases. By fitting this to
+a curve, we can extrapolate to the zero-noise limit and obtain a better estimate.
+
+Below we execute the folded circuits using the executor function defined at the start of this example.
+
+```{code-cell} ipython3
+expectation_values = [braket_ionq_execute(c) for c in folded_circuits]
+print(f"Expectation values:\n{expectation_values}")
+```
+
+**Note:** Using a _batched_ executor could speedup this step, as explained in the [Executors](../guide/executors.myst) section.
```

Comment:
```suggestion
```{note}
Using a _batched_ executor could speed up this step, as explained in the [Executors](../guide/executors.myst) section.
```
```

Reply:
Suggestion to use the block-style notes as in the "Frontends and Backends" section of the user guide.